benchs/tpch/run.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -eu
+set -o pipefail
+
+TAR_VER=$(tarantool -v | grep -e "Tarantool" |  grep -oP '\s\K\S*')
+numaopts="numactl --membind=1 --cpunodebind=1 --physcpubind=6,7,8,9,10,11"
+base_dir=$(pwd)
+
+cd /opt/tpch
+make TPC-H.db
+
+killall tarantool tpcc_load 2>/dev/null || true
+sync && echo "sync passed" || echo "sync failed with error" $?
+echo 3 > /proc/sys/vm/drop_caches
+
+$numaopts /opt/tpch/bench_queries.sh 2>&1 | tee bench-sqlite.log
+make 00000000000000000000.snap
+
+killall tarantool tpcc_load 2>/dev/null || true
```

Comment:
The same here - `tpcc_load` appears to be irrelevant. If not intentional.

Reply:
it was copy-paste. removing